docs/circuit_cutting/cutqc/index.rst

```diff
@@ -6,14 +6,8 @@ The ``cutqc`` module implements the wire cutting technique and
 automatic cut finding method described in the research paper
 `arXiv:2012.02333 <https://arxiv.org/abs/2012.02333>`_.  Historically,
 this was the original circuit cutting implementation in the Circuit
-Knitting Toolbox.  Going forward, this module is considered legacy
-code that is supported, but new features are not expected.  New users
-of the toolbox should use the new circuit cutting interface, instead,
-unless they specifically need a feature that is not (yet) supported by
-the new code.  These features currently specific to ``cutqc`` include:
-
-- Reconstruction of probability distributions (rather than expectation values)
-- Automatic cut finding
+Knitting Toolbox.  Going forward, this module is deprecated.  New users
```

Comment:
Splitting hairs but since this applies to _all_ users, we may want to say: 
```suggestion
Knitting Toolbox.  Going forward, this module is deprecated.  Users
```